clusterloader2/pkg/prometheus/manifests/dashboards/dns.dashboard.py

```diff
@@ -44,10 +44,12 @@
         title="probe: lookup rate",
         targets=[
             g.Target(
+                datasource = "$source",
```

Comment:
Why do we need to set it separately?

Reply:
d.Target could handle source, d.TargetWithIntervalSet (naming is hard) could handle interval